bindings/python/dlite-entity-python.i

```diff
@@ -158,11 +158,12 @@ def get_instance(id: "str", metaid: "str" = None, check_storages: "bool" = True)
 %extend _DLiteProperty {
   %pythoncode %{
     def __repr__(self):
+        ref = ', ref=%r' % self.ref if self.ref else ''
         shape = ', shape=%r' % self.shape.tolist() if self.ndims else ''
         unit = ', unit=%r' % self.unit if self.unit else ''
         descr = ', description=%r' %self.description if self.description else ''
-        return 'Property(%r, type=%r%s%s%s)' % (
-            self.name, self.type, shape, unit, descr)
+        return 'Property(%r, type=%r%s%s%s%s)' % (
```

Comment:
How does this work? If I understand it correctly,  everything except self.name passed to the type argument. Why?

Reply:
Since all arguments, except `name` and `type` are optional, we wither want to include them in the returned `repr()` representation as strings like 

    ", shape=['N', 'N+1']" 

if they are set or the empty string "" if they are unset (scalar, in case of shape). 

Are the variable names (`ref`, `shape`, `unit` and `descr`) are misleasing?